src/contextTypes.ts

```diff
@@ -79,6 +79,10 @@ export interface MosaicWindowActions {
    * Sets the open state for the tray that holds additional controls
    */
   setAdditionalControlsOpen: (open: boolean) => void;
+  /**
+   * Sets the open state for the tray that holds additional controls
+   */
+  toggleAdditionalControlsOpen: () => void;
```

Comment:
the "open" at the end might imply that we want to open the tray, maybe just "toggle" is fine? 

```suggestion
  /**
   * Opens/closes the tray that holds additional controls
   */
  toggleAdditionalControls: () => void;
```